<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <!--Robot 1 Description-->
  <arg name="robot" default="puzzlebot"/>


  <!-- Load controller configurations from YAML file to parameter server -->
  <rosparam file="$(find puzzlebot_control)/config/puzzlebot_diff_control.yaml" command="load" />
  <rosparam file="$(find puzzlebot_control)/config/puzzlebot_gazebo_control.yaml" command="load" />
  <rosparam file="$(find puzzlebot_control)/config/puzzlebot_joint_states.yaml" command="load" />
    
  <!--Spawn controller -->
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"
    args="joint_state_controller base_controller">
  </node>

  <node name="wheel_vel" pkg="puzzlebot_control" type="wheel_vel.py"  output="screen"/>

</launch>